charts/px-backup/templates/px-backup/pxcentral-etcd.yaml

```diff
@@ -6,10 +6,10 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: px-backup-etcd-scripts
+  name: pxc-backup-etcd-scripts
```

Comment:
I would prefer to keep `pxc` reserved for all the `pxc` (The upcoming PX CLI) project.

Since both of them will be used in the same cluster, best to have separate naming convention.

You can use `pxcentral` instead.

Reply:
Yeah, we changed all components prefix to pxcentral but for etcd, we had an issue in migrating existing ETCD to chart one where endpoints are changed so reverted change for etcd and added `pxc` as prefix to support operator install to chart upgrade support.